CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -1111,7 +1122,16 @@ local function CreateSettingsHub()
     end)
 
   -- Dev Console Connections
-  ContextActionService:BindCoreAction(DEV_CONSOLE_ACTION_NAME, toggleDevConsole, false, Enum.KeyCode.F9)
+  ContextActionService:BindCoreAction(DEV_CONSOLE_ACTION_NAME, 
+    toggleDevConsole, 
+    false,
+    Enum.KeyCode.F9)
+
+  -- Quick Profiler connections
+  ContextActionService:BindCoreAction(QUICK_PROFILER_ACTION_NAME, 
+    toggleQuickProfiler, 
+    false,
+    Enum.KeyCode.F7)
```

Comment:
F7 is in the client code right now as being able to toggle UI on/off. Can we switch this to Shift+F7? should be able to just add a check in toggleQuickProfiler: game:GetService("UserInputService"):IsKeyDown(Enum.KeyCode.LeftShift) or RightShift or both.